src/functions/operators.js

```diff
@@ -59,7 +67,16 @@ defineFunction({
 
             let word = "";
             for (let i = 0; i < temp.length; i++) {
-                word += temp[i].children[0].text;
+                if (temp[i].type === "mi") {
+                    word += temp[i].children[0].text;
+                } else if (temp[i].type === "mspace") {
+                    // TODO(kevinb) differentiate space width
+                    word += " ";
+                } else {
+                    const pos = temp[i].lexer.pos;
+                    throw new Error(
+                        `unrecognized group in \\operatorname at ${pos}`);
```

Comment:
Does it make sense to raise a `ParseError` inside a MathML builder?  It's not really a parser error, it's just something we don't know how to deal with.